frontend/src/employee-frontend/components/unit/unit-reservations/StaffAttendanceTable.tsx

```diff
@@ -564,14 +571,21 @@ const AttendanceRow = React.memo(function AttendanceRow({
                 <AttendanceTimes data-qa="attendance-day">
                   {attendancesForToday.length > 0 ? (
                     attendancesForToday.map((attendance, i) => (
-                      <AttendanceCell key={i}>
-                        <AttendanceTime data-qa="arrival-time">
-                          {renderTime(attendance.arrived, date)}
-                        </AttendanceTime>
-                        <AttendanceTime data-qa="departure-time">
-                          {renderTime(attendance.departed, date)}
-                        </AttendanceTime>
-                      </AttendanceCell>
+                      <Tooltip
+                        key={i}
+                        tooltip={attendanceTooltipText(attendance)}
+                        data-qa="attendance-tooltip"
+                      >
+                        <AttendanceCell>
+                          <AttendanceTime data-qa="arrival-time">
+                            {renderTime(attendance.arrived, date)}
+                          </AttendanceTime>
+                          <AttendanceTime data-qa="departure-time">
+                            {renderTime(attendance.departed, date)}
+                            {attendance.departedAutomatically ? `(*)` : ''}
+                          </AttendanceTime>
+                        </AttendanceCell>
+                      </Tooltip>
                     ))
```

Comment:
```
                    attendancesForToday.map((attendance, i) => {
                      const tooltip = attendanceTooltipText(attendance)
                      return (
                        <Tooltip
                          key={i}
                          tooltip={tooltip}
                          data-qa="attendance-tooltip"
                        >
                          <AttendanceCell>
                            <AttendanceTime data-qa="arrival-time">
                              {renderTime(attendance.arrived, date)}
                            </AttendanceTime>
                            <AttendanceTime data-qa="departure-time">
                              {renderTime(attendance.departed, date)}
                              {tooltip && ' (*)'}
                            </AttendanceTime>
                          </AttendanceCell>
                        </Tooltip>
                      )
                    })
```
This way you aren't duplicating logic and adding another tooltip reason only requires changing the tooltip returning function.